.rfc/RFC0000-rfc-process.md

```diff
@@ -0,0 +1,156 @@
+---
+RFC:          RFC0000
+Author:       Mikey Lombardi
+Status:       Draft
+SupercededBy: N/A
+Version:      1.0
+Area:         Process
+---
+
+# DSC RFC Process and Guidelines
+
+A DSC RFC (Request for Comments) is a publication to propose design changes and improvements to
+DSC. This provides the community an opportunity to provide feedback before code is written where it
+becomes harder to change at the risk of compatibility.
+
+This process was adapted from the PowerShell RFC process, which itself was adapted from the Chef
+RFC process and the DMTF.org process.
+
+## Roles
+
+- **Author**: All members of the community are allowed to author new RFCs and can provide feedback
+  to any RFC.
+- **DSC Committee**: The design committe that votes to accept or reject an RFC. Currently, the DSC
```

Comment:
Since DSC team members are also members of the DSC WG, I wonder if we need to have a Committee above it

Reply:
I'm in favor of streamlining this to only including the working group for the time being - if we need to add a higher layer later, we can always amend the process for that.

Reply:
Updated.